### PR TITLE
Error handling with Ruby on Rails

### PR DESCRIPTION
**Pull request by ledermann**
Code snippets to support exception handling in Rails. 
Tested in https://github.com/ledermann/pingcrm/

### CODE REVIEW

**Comment by reinink** [comment]
Ping @bknoles and @BrandonShar. Can you please review this before I merge it in?

File: pages/error-handling.mdx [hunk 1]
```diff
@@ -56,7 +56,27 @@ In production you'll want to return a proper Inertia error response instead of r
...
+              status: request.path_info[1..-1].to_i
+            }
+          end
+        end
+        
+        # config/application.rb
+        module MyApplication
+          class Application < Rails::Application
+            # ...
+            config.exceptions_app = ->(env) { ExceptionsController.action(:show).call(env) }
```
Comment (BrandonShar) [inline pages/error-handling.mdx]: Is it possible to return the inertia render directly from here? (Genuinely asking, I'm not sure) That would be most similar to the Laravel example.

Comment (ledermann) [reply pages/error-handling.mdx]: Hm, we could use an anonymous controller class:

```ruby
config.exceptions_app = ->(env) do
  Class.new(ActionController::Base) do
    def show
      render inertia: 'Error', props: {
        status: request.path_info[1..-1].to_i
      }
    end
  end.action(:show).call(env)
end
```

Everything in one place, but still a bit different from the Laravel example. If you prefer, I'll change the PR.

**Comment by claudiodekker** [comment]
Thanks for contributing to our documentation!

Unfortunately, I'm going to delay merging this code for now, given that the documentation was private for quite some time while they were being updated. As a result, it's now difficult to judge what's still relevant/up-to-date, and what isn't, so we're mass-closing older PR's with this message (sorry!)

Feel free to update and re-submit your change if you believe that it's still relevant, at which point we'll reconsider adding this (assuming any previous views discussed in this thread have changed).

Thanks!

**Pull request closed without merging**